[2022-02-21 18:30:26,499] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-02-21T18:30:23.179955+00:00 [queued]>
[2022-02-21 18:30:26,510] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-02-21T18:30:23.179955+00:00 [queued]>
[2022-02-21 18:30:26,511] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 18:30:26,511] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-02-21 18:30:26,511] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 18:30:26,522] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): enter_virtual_env> on 2022-02-21T18:30:23.179955+00:00
[2022-02-21 18:30:26,526] {standard_task_runner.py:52} INFO - Started process 46239 to run task
[2022-02-21 18:30:26,533] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'enter_virtual_env', '2022-02-21T18:30:23.179955+00:00', '--job-id', '45', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmpi8l65j28', '--error-file', '/tmp/tmplwurc3hg']
[2022-02-21 18:30:26,534] {standard_task_runner.py:77} INFO - Job 45: Subtask enter_virtual_env
[2022-02-21 18:30:26,574] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.enter_virtual_env 2022-02-21T18:30:23.179955+00:00 [running]> on host 531edfb863fc
[2022-02-21 18:30:26,609] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=enter_virtual_env
AIRFLOW_CTX_EXECUTION_DATE=2022-02-21T18:30:23.179955+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-21T18:30:23.179955+00:00
[2022-02-21 18:30:26,610] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-02-21 18:30:26,611] {bash.py:158} INFO - Running command: . /app/env/bin/activate
[2022-02-21 18:30:26,621] {bash.py:169} INFO - Output:
[2022-02-21 18:30:26,627] {bash.py:177} INFO - Command exited with return code 0
[2022-02-21 18:30:26,647] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=enter_virtual_env, execution_date=20220221T183023, start_date=20220221T183026, end_date=20220221T183026
[2022-02-21 18:30:26,671] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-21 18:30:26,702] {local_task_job.py:146} INFO - Task exited with return code 0
